symbology/custom-dictionary-style/README.md

```diff
@@ -0,0 +1,36 @@
+# Custom dictionary style
+
+Use a custom dictionary style (.stylx) to symbolize features using a variety of attribute values.
+
+![Custom dictionary style](CustomDictionaryStyle.png)
+
+## Use case
+
+When symbolizing geoelements in your map, you may need to convey several pieces of information with a single symbol. You could try to symbolize such data using a unique value renderer, but as the number of fields and values increases, that approach becomes impractical. With a dictionary renderer you can build each symbol on-the-fly, driven by one or more several attribute values, and handle a nearly infinite number of unique combinations.
```

Comment:
Either "one or more" or "several" attribute values. Not both.

Reply:
Good spot, thanks